### PR TITLE
e2e tests for machinehealthchecks

### PR DESCRIPTION
**Pull request by yithian**
for OSD-3901

### CODE REVIEW

File: pkg/e2e/osd/machinehealthcheck.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+				),
+			))
+		}
+	})
+
+	ginkgo.It("should replace unhealthy nodes", func() {
+		r := h.Runner("chroot /host -- systemctl stop kubelet")
+		r.Name = "stop-kubelet"
+		// i can't believe SecurityContext.Privileged is a pointer to a bool
+		thisIsSoDumb := true
```
Comment (jeefy) [inline pkg/e2e/osd/machinehealthcheck.go:58]: Nit: I can't believe it either, but in cases like this I typically name the variable `truePointer` ;)

Comment (yithian) [reply pkg/e2e/osd/machinehealthcheck.go]: that's a much more descriptive variable name that still maintains the silliness, changed :)

**Comment by cblecker** [comment]
LGTM

**Pull request merged**